.github/workflows/deploy.yml

```
@@ -74,7 +74,10 @@ jobs:
 
             if [ "${PASSED:-0}" -ne 1 ]; then
               echo "헬스체크 실패 → 새 컨테이너 로그"
-              sudo docker logs "$NEW" | tail -n 200
+              sudo docker logs "$NEW_NAME" | tail -n 200
+              echo "새 컨테이너 정리"
+              docker stop "$NEW_NAME" || true        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)